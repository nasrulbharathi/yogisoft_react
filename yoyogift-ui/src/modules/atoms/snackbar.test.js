import React from "react";
import { shallow } from "enzyme";
import MySnackbar from "./Snackbar";

describe("Snackbar", () => {
  it("should be defined", () => {
    expect(MySnackbar).toBeDefined();
  });
  it("should render Snackbar", () => {
    const component = shallow(<MySnackbar />);
    expect(component).toMatchSnapshot();
  });
});
