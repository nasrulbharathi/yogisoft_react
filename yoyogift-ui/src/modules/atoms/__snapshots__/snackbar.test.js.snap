// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snackbar should render Snackbar 1`] = `ShallowWrapper {}`;
